src/lib/localStorage.js

```diff
@@ -0,0 +1,19 @@
+import getToken from './tokens';
+
+export function checkForToken(key, value) {
+  try {
+    const saveToken = window.localStorage.getItem(key);
+    return saveToken ? JSON.parse(saveToken) : value;
+  } catch (error) {
+    console.log(error);
+    return value;
+  }
+}
+
+export function addTokenToLocalStorage(key) {
+  try {
+    window.localStorage.setItem(key, JSON.stringify(getToken()));
+  } catch (error) {
+    console.log(error);
+  }
+}
```

Comment:
First of all, awesome!! I love how you extracted these functions and included catch blocks.

This is the TINIEST thought. I really like how you abstracted these functions out so that they could be used to check for other key values in localStorage, but some of the naming is still specific to tokens. **IF** your goal is to really make these functions abstracted for a general use case, you might consider changing the function names and variable names to reflect that. Ex: `checkLocalStorageForKey` or `addKeyValuePairToLocalStorage`. 

On the opposite spectrum, I might update to argument `value` to be `defaultValue` to make it a little more clear to other developers what its purpose is.

You definitely don't have to take these suggestions, and @yenly or @skylerwshaw might disagree. Again, awesome work!

Reply:
I think you make a really good point, @laurenyz! Thank you!

Reply:
👍 also feel free to change those names to something more succinct 😂

Reply:
Great job on abstracting out to helper functions and minding potential errors. Naming things are difficult, and however specific you choose is good.